servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -359,6 +394,13 @@ private FileInfo getFileInfoInternal(Inode inode) throws FileDoesNotExistExcepti
   }
 
   /**
+   * @return a read-only view of the inode tree
```

Comment:
Ideally it could be a view InodeTree, but (1) `FileStore` has method of getFileInfo, which is method defined in `FileSystemMaster` (2) since `FileInfo` is returned not `Inode`, a view files might be more accurate. But I'm open to suggestions and I also like the view to correspond to a concrete class.